### ISSUE TITLE
Version 1.1.0 optional chaining operator ( ?. ) make error "Unexpected token '.' "

### ISSUE THREAD

**Issue by TIS-OMiddle** [siddharthkp/babel-plugin-open-source#13]
```
let editor = state.opts?.editor?.toLowerCase() || 'vscode';
                              ^

SyntaxError: Unexpected token '.'
    at compileFunction (<anonymous>)
```

🤔  Maybe we should avoid using optional chaining in `babel.js` or use babel to compile `babel.js` before publish?

**Comment by siddharthkp** [comment]
Just checking, which node version are you on?

**Comment by TIS-OMiddle** [comment]
> Just checking, which node version are you on?

node v12.19.0, npm 7.22.0. Project is created by `react-app-rewired` with this code: `addBabelPlugin('babel-plugin-open-source')`

**Comment by siddharthkp** [comment]
Node 12 is fair game! Would you like to create a PR to not use optional chaining operator?

**Comment by siddharthkp** [comment]
Closed in https://github.com/siddharthkp/babel-plugin-open-source/issues/13

Released in 1.1.1
